frontend/src/components/Sidebar/DistrictMap.tsx

```diff
@@ -65,17 +66,20 @@ export default function DistrictMap(): JSX.Element {
     if (fixedLegendMaxValue) {
       return fixedLegendMaxValue;
     }
-    let max = 0;
+    let max = 1; // if min of legend is 0, then default max is 1
```

Comment:
Yes, the menus look different because of the different Amchart library versions. So what shall we do in this case? Move the line chart to amchart5?

Regarding pdf export for district map, it is working on my end. See attached.
[chart (1).pdf](https://github.com/DLR-SC/ESID/files/9682278/chart.1.pdf)